Scripts/GameSaver.gd

```diff
@@ -14,6 +14,7 @@ onready var game_data = get_parent().get_node("GameData")
 onready var level_manager = get_parent().get_node("LevelManager")
 onready var elf_stats = get_parent().get_node("ElfStats")
 onready var elf = get_parent().get_node("Elf")
+onready var publician = get_parent().find_node("Publician")
```

Comment:
Little typo: should be "publican" without second 'i' (in find_node change node name as well)

Reply:
Oh yes